vendor/engines/journal/lib/weby/components/newsletter/assets/stylesheets/newsletter.css.scss

```diff
@@ -0,0 +1,7 @@
+.newsletter_component {
+  max-width: 300px;
+  input#email {
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid using id selectors<br>Avoid qualifying id selectors with an element.